fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1257,8 +1258,16 @@ void continuationDuringRepartitioningTest(boolean isGrouped,
         assertEquals(docCount, partitionInfos.get(0).getCount());
 
         byte[] continuation;
-        Sort sort = sortType == SortType.UNSORTED ? null :
-                    new Sort(new SortField(isSynthetic ? "complex_timestamp" : "timestamp", SortField.Type.LONG, sortType == SortType.DESCENDING));
+        final Sort sort;
+        if (sortType == SortType.UNSORTED) {
+            sort = null;
+        } else if (uniqueTimestamps) {
+            sort = new Sort(new SortField(isSynthetic ? "complex_timestamp" : "timestamp", SortField.Type.LONG, sortType == SortType.DESCENDING));
+        } else {
+            sort = new Sort(new SortField(isSynthetic ? "complex_timestamp" : "timestamp", SortField.Type.LONG, sortType == SortType.DESCENDING),
+                    new SortField(LuceneIndexMaintainer.PRIMARY_KEY_SEARCH_NAME, SortField.Type.STRING, sortType == SortType.DESCENDING));
```

Comment:
I think you mentioned this offline, but can we make this implicit?
If we can, we should probably test the possibility that the user also specifies the primary key in the search.

Reply:
the implicit logic would then be:
1. if `Sort` has `partitionFieldName` first and doesn't have `_s` second, insert `_s` into the sort fields immediate after the first (even if it means displacing potentially other fields to the right)